client/me/account-password/index.jsx

```diff
@@ -90,7 +92,7 @@ module.exports = React.createClass( {
 					debug( 'Error saving password: ' + JSON.stringify( error ) );
 
 					// handle error case here
-					notices.error( this.translate( 'There was a problem saving your password. Please, try again.' ) );
+					this.props.errorNotice( this.translate( 'There was a problem saving your password. Please, try again.' ) );
```

Comment:
I tried this, but apparently this violates eslint `no-redeclare` rule.
We imported `errorNotice` to provide it to connect higher in the scope.
I have no sensible solution for this. If we rename it, we cannot use destructuring.